openmls/src/schedule/mod.rs

```diff
@@ -1190,7 +1184,7 @@ impl EpochSecrets {
         let secret_tree = self.encryption_secret.create_secret_tree(treesize);
         (
             GroupEpochSecrets {
-                init_secret: self.init_secret,
+                init_secret: Some(self.init_secret),
```

Comment:
I missed this before: There's probably also no reason to keep the `Option` in `GroupEpochSecrets`.